class.jetpack.php

```diff
@@ -560,6 +564,19 @@ private function __construct() {
 		}
 	}
 
+	function rest_api_init() {
+		register_rest_route( 'jetpack/v1', '/verify_registration', array(
+			'methods' => 'POST',
+			'callback' => array( $this, 'rest_verify_registration' ),
+		) );
```

Comment:
It would probably be worthwhile in the declaration here to list out the parameters that the endpoint accepts via an `args` parameter.

http://v2.wp-api.org/extending/adding/#arguments